src/_guides/technical-writing/5-improvements-to-openapi-operation-documentation.md

```diff
@@ -0,0 +1,255 @@
+---
+title: 5 Improvements to OpenAPI Operation Documentation
+authors: james
+canonical_url: 
+excerpt: A look at five operation-specific improvements that you can make to deliver on a great developer experience while reducing the support costs of your API. 
+---
+
+After conducting API design and documentation reviews for over a decade, there are common mistakes that most developers make when composing their reference documentation using the OpenAPI Specification. Let’s look at five operation-specific improvements that you can make to deliver on a great developer experience while reducing the support costs of your API. 
+
+## 1. Expand Your Operation Descriptions:
+
+Often, API documentation is left to the end of the schedule, resulting in a rushed set of reference documentation. The result are terse operation details that state the obvious and leave out the specifics of your well-designed and implemented API operation. 
+
+### Example of an Incomplete Operation Documentation
+
+For example, an operation might be described simply as `GET /users` with no further explanation, leaving users to guess what the endpoint does.
+
+**Example Mistake:**
+```yaml
+paths:
+  /users:
+    get:
+      summary: "Get users"
+      responses:
+        200:
+          description: "OK"
+```
+
+### Improved Version:
+Detailed and clear descriptions enhance understanding and usability. The improved version should thoroughly explain the operation, including its purpose, parameters, and response types.
+
+**Improved Example:**
+```yaml
+paths:
+  /users:
+    get:
+      summary: "Retrieves a paginated list of users"
+      description: >
+        This operation returns a paginated list of users registered in the system, 
+        including their usernames and contact information. Use query parameters 
+        for filtering and sorting the result set.
+      parameters:
+        - in: query
+          name: limit
+          schema:
+            type: integer
+            default: 10
+          description: "Limit the number of users returned. Defaults to 10 if not specified"
+      responses:
+        200:
+          description: "A paginated list of users with detailed information"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserList'
+```
+
+## 2. Add Examples to Your API Operations and Schema
+
+Ever since OpenAPI v3, examples have been built-in to the specification. OpenAPI examples are used to replace the default values generated by documentation rendering frameworks, such as SwaggerUI. They offer the API provider the ability to customize the examples generated by these documentation frameworks to make it easier to understand your API operation. 
```

Comment:
Might be nice to self-promote here. Not strong opinion but if we mention Swagger UI, we can probably mention Bump too :)

```suggestion
Ever since OpenAPI v3, examples have been built-in to the specification. OpenAPI examples are used to replace the default values generated by documentation rendering frameworks, such as SwaggerUI or Bump.sh. They offer the API provider the ability to customize the examples generated by these documentation frameworks to make it easier to understand your API operation. 
```